contrib/runsim/notification.go

```diff
@@ -20,61 +19,64 @@ const (
 	awsRegion       = "us-east-1"
 )
 
-var (
-	simTimeStamp = time.Now().Format("01-02-2006_15:05:05")
-)
-
-func awsErrHandler(err error) {
+func awsErrHandler(err error) error {
 	if awsErr, ok := err.(awserr.Error); ok {
 		switch awsErr.Code() {
```

Comment:
do we need this `switch` statement?

Reply:
Not right now, but I want it in there for later when I'll get around to handling some of these error codes better.

Reply:
I'd delete it and re-add later it if necessary